_overviews/jdk-compatibility/overview.md

```diff
@@ -13,18 +13,25 @@ permalink: /overviews/jdk-compatibility/:title.html
 
 Scala runs primarily on the Java Virtual Machine (JVM). As Scala and the JVM improve independently over time, Scala drops compatibility with older versions of the Java Developer Kit (JDK) in order to focus development efforts on supporting new JVM features that benefit Scala.
 
-This table shows the first Scala release in each series that functions on each JDK.
+This table shows the first Scala release in each series that functions on each JVM release.
 
-| JDK version | First Scala release supporting JDK version per series |
+| JVM version | First Scala compiler release supported to run on this JVM |
 |:-----------:|:-----------------------------------------------------|
 | 9           | 2.12.4, 2.11.12, 2.10.7                              |
 | 8           | 2.12.0, 2.11.0, 2.10.0                               |
 | 7           | 2.11.0, 2.10.0                                       |
 | 6           | 2.11.0, 2.10.0                                       |
 
+### Running versus compiling and required / supported JVM
+For most users, we recommend using Java 8 for *compiling* (and running) Scala code. Since the JVM is backward compatible, it is usually safe to use a newer JVM to *run* your code compiled by the Scala compiler for older JVM versions. There are notable exceptions with experimental/unsafe features, and the introduction of the module system in Java 9. The Scala compiler does usually need updates to run properly on newer versions of the JVM, so make sure to use the appropriate JVM when compiling your code.
```

Comment:
This limits the user from being able to call Java 9 APIs. Perhaps we should be explicit about that restriction.